clients/src/test/java/org/apache/kafka/common/serialization/SerializationTest.java

```diff
@@ -147,6 +147,13 @@ public void floatSerdeShouldPreserveNaNValues() {
         }
     }
 
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
Can we add a unit test for serialization as well, and maybe one for "normal" deserialization (where no exception is thrown)?

Reply:
Fixed. Two tests added - `testSerializeVoid`, `testDeserializeVoid`